analyses/tmb-compare-tcga/README.md

```diff
@@ -16,11 +16,15 @@ This analysis compares the Pediatric Brain Tumor samples of this dataset to the
 
 ## Usage
 
-To run this from the command line, use:
+To run this from the command line, first, the `snv-caller` scripts: `snv-caller/run_caller_consensus_analysis-tcga.sh` and `snv-caller/run_caller_consensus_analysis-pbta.sh` must be run.  
```

Comment:
Perhaps also add the command that can be used to run the mentioned scripts here, this could be helpful for someone who is not familiar with bash _or_ better yet, you can link to the `How to run caller consensus analysis` section of `snv-callers/README.md` for this.

Reply:
Added a link!